topics/icrc_25_signer_interaction_standard.md

```diff
@@ -38,8 +38,8 @@ The purpose of the `permission` messages is to establish a connection between a
 - `name` (`text`, optional): An optional user-friendly name of the network.
 - `rpcUrl` (`text`, optional): An optional custom RPC URL associated with the network.
 
-`scopes`: A list of permission scopes the relying party requires. If the signer does not support any of the requested scopes, it should ignore it. Possible values: 
-- `"canister_call"`
+`scopes`: A list of permission scope objects the relying party requires. If the signer does not support any of the requested scopes, it should ignore it. Permission scope properties:
```

Comment:
This is not about the change in this PR, but in the sentence "If the signer does not support any of the requested scopes, it should ignore it." the final "it" that should be ignored is not clear: should the signer ignore the request (i.e. not reply at all)?, or should the signer reply with an error?

Reply:
Thanks, I reformulated the sentence.